apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/start/en.ftl

```diff
@@ -1,9 +1,8 @@
 ## Page
 checkout-signin-or-create = 1. Sign in or create a { -product-mozilla-account }
 checkout-create-account = Create a { -product-mozilla-account }
-next-continue-with-google-button = Continue with { -brand-google }
-next-continue-with-apple-button = Continue with { -brand-apple }
-next-payment-confirm-with-legal-links-static-3 = I authorize { -brand-mozilla } to charge my payment method for the amount shown, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
+continue-with-google-button = Continue with { -brand-google }
```

Comment:
Ah, I thought we could reuse the `continue-with-google-button` and `continue-with-apple-button` ftl ids that are in fxa-settings.

Reply:
If these string IDs already exist in another file these should be re-named to something to avoid having two strings with the same ID (even if they are in different files).